### PR TITLE
unreal: clarify environment defaults

### PR DESCRIPTION
**Pull request by bruno-garcia**
Reodered things a bit and added the values mentioned by @tustanivsky on the linked ticket

Resolves:
* https://github.com/getsentry/sentry-docs/issues/11056

### CODE REVIEW

File: docs/platforms/unreal/configuration/environments.mdx [hunk 1]
```diff
@@ -5,10 +5,13 @@ sidebar_order: 30
 ---
 
 Environments tell you where an error occurred, whether that's in your production system, your staging server, or elsewhere.
+It help you better filter issues, releases, and user feedback in the Issue Details page of sentry.io, which you learn more about in our [documentation that covers using environments](/product/sentry-basics/environments/).
 
 Sentry automatically creates an environment when it receives an event with the `environment` parameter set.
 
-Environments are case-sensitive. The environment name can't contain newlines, spaces or forward slashes, can't be the string "None", or exceed 64 characters. You can't delete environments, but you can [hide](/product/sentry-basics/environments/#hidden-environments) them.
+By default, the SDK reports `Editor` when running inside the Unreal Engine Editor. 
+For shipping builds it'll use `Release` and in other configurations it'll use `FApp::GetBuildConfiguration()`
```
Comment (tustanivsky) [inline docs/platforms/unreal/configuration/environments.mdx:13]:
```suggestion
For shipping builds it'll use `Release` and in other configurations it'll use `FApp::GetBuildConfiguration()`.
```

File: docs/platforms/unreal/configuration/environments.mdx [hunk 2]
```diff
@@ -5,10 +5,13 @@ sidebar_order: 30
 ---
 
 Environments tell you where an error occurred, whether that's in your production system, your staging server, or elsewhere.
```
Comment (lizokm) [inline docs/platforms/unreal/configuration/environments.mdx:7]:
```suggestion
Environments in Sentry let you know where an error occurred, (such as in production, staging server, or another location).
```

File: docs/platforms/unreal/configuration/environments.mdx [hunk 3]
```diff
@@ -5,10 +5,13 @@ sidebar_order: 30
...
+It help you better filter issues, releases, and user feedback in the Issue Details page of sentry.io, which you learn more about in our [documentation that covers using environments](/product/sentry-basics/environments/).
```
Comment (lizokm) [inline docs/platforms/unreal/configuration/environments.mdx:8]:
```suggestion
They make it easier to filter issues, releases, and user feedback on the Issue Details page, (which you can navigate to by clicking any issue on the [**Issues**](https://sentry.io/orgredirect/organizations/:orgslug/issues/) page). To learn more, read our docs about [using environments](/product/sentry-basics/environments/).
```

File: docs/platforms/unreal/configuration/environments.mdx [hunk 4]
```diff
@@ -26,6 +29,4 @@ USentrySubsystem* SentrySubsystem = GEngine->GetEngineSubsystem<USentrySubsystem
 SentrySubsystem->InitializeWithSettings(SettingsDelegate);
 ```
 
-By default, the SDK reports `Editor` when running inside the Unreal Engine Editor. You can also set the default environment to `Release`, `Development`, or `Debug` in your build configuration or override the environment value manually.
-
-Environments help you better filter issues, releases, and user feedback in the Issue Details page of sentry.io, which you learn more about in our [documentation that covers using environments](/product/sentry-basics/environments/).
+Environments are case-sensitive. The environment name can't contain newlines, spaces or forward slashes, can't be the string "None", or exceed 64 characters. You can't delete environments, but you can [hide](/product/sentry-basics/environments/#hidden-environments) them.
```
Comment (lizokm) [inline docs/platforms/unreal/configuration/environments.mdx:32]:
```suggestion
**Important things to note about environments:**

- Environment names are case-sensitive.
- Names cannot include new lines, spaces, or forward slashes.
- Certain strings, like "None", aren't allowed.
- The maximum length for an environment name is 64 characters.
- Environments can't be deleted, but can be [hidden](/product/sentry-basics/environments/#hidden-environments) if no longer needed.

```

File: docs/platforms/unreal/configuration/environments.mdx (hunk 1 above)
Comment (lizokm) [inline docs/platforms/unreal/configuration/environments.mdx:13]:
```suggestion
When running inside the Unreal Engine Editor, the SDK reports the environment as `Editor`. For shipping builds, it defaults to `Release`. In other configurations it uses the `FApp::GetBuildConfiguration()`. 
```

File: docs/platforms/unreal/configuration/environments.mdx [hunk 5]
```diff
@@ -5,10 +5,13 @@ sidebar_order: 30
...
 
 Environments tell you where an error occurred, whether that's in your production system, your staging server, or elsewhere.
+It help you better filter issues, releases, and user feedback in the Issue Details page of sentry.io, which you learn more about in our [documentation that covers using environments](/product/sentry-basics/environments/).
 
 Sentry automatically creates an environment when it receives an event with the `environment` parameter set.
 
-Environments are case-sensitive. The environment name can't contain newlines, spaces or forward slashes, can't be the string "None", or exceed 64 characters. You can't delete environments, but you can [hide](/product/sentry-basics/environments/#hidden-environments) them.
+By default, the SDK reports `Editor` when running inside the Unreal Engine Editor. 
+For shipping builds it'll use `Release` and in other configurations it'll use `FApp::GetBuildConfiguration()`
+You can also set the default environment to `Release`, `Development`, or `Debug` in your build configuration or override the environment value programatically.
```
Comment (lizokm) [inline docs/platforms/unreal/configuration/environments.mdx:14]:
```suggestion
You can also set the default environment to `Release`, `Development`, or `Debug` in your build configuration, or override it programmatically.
```

**Review by lizokm — APPROVED** [review]
Thanks for updating this! I've left some wording suggestions.

**Comment by bruno-garcia** [comment]
> Thanks for updating this! I've left some wording suggestions.

oh no I merged this on mobile and didn't realize the suggestions. I'm on pto will make the fixes once I'm back.

**Pull request merged**